RELEASE.md

```diff
@@ -25,3 +25,46 @@
 6. Announce the release on the [mailing list](https://groups.google.com/g/jaeger-tracing), [slack](https://cloud-native.slack.com/archives/CGG7NFUJ3), and [twitter](https://twitter.com/JaegerTracing?lang=en).
 
 Maintenance branches should follow naming convention: `release-major.minor` (e.g.`release-1.8`).
+
+## Release managers
+
+A Release Manager is the person responsible for ensuring that a new version of Jaeger is released. This person will coordinate the required changes, including to the related components such as UI, IDL, and jaeger-lib and will address any problems that might happen during the release, making sure that the documentation above is correct.
+
+In order to ensure that knowledge about releasing Jaeger is spread among maintainers, we rotate the role of Release Manager among maintainers.
+
+Here are the release managers from past and future versions:
+
+| Version   | Release Manager  |
```

Comment:
Is it important to list all past release managers? Is this mainly just meant to give credit to the person doing the work? 

Feels like just one more thing to do during release :)

Reply:
Great point. I originally just wanted to share the past data via Slack, to justify the order for the future releases but thought it could be a good idea to have it listed somewhere. I'm fine with removing it.

Reply:
I agree with Joe, no need to list historical data. Going forward the table will get updated with future releases. It won't be necessarily done during the release

Reply:
Alright, I removed it.